### ISSUE TITLE
is version 2.1.0-SNAP2 a snapshot?

### ISSUE THREAD

**Issue by gzm55** [scalatest/scalatest-maven-plugin#94]
two strange version 2.1.0-SNAP1 and 2.1.0-SNAP2 are released to [central](https://search.maven.org/artifact/org.scalatest/scalatest-maven-plugin/2.1.0-SNAP2/maven-plugin).

@cheeseng @bvenners Are these versions are release or snapshot?

**Comment by gzm55** [comment]
are all versions like *-SNAPx snapshot?

**Comment by cheeseng** [comment]
@gzm55 Yes.

**Comment by gzm55** [comment]
OK. since they are pushed to the release central repo, it would be better to describe this version rule on the readme.

**Comment by cheeseng** [comment]
@gzm55 Good idea, @bvenners had a good explanation and reason behind it, I'll have to get it back to put it on README.
